lib/models/samples_widget_list.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:arcgis_maps_sdk_flutter_samples/samples/apply_unique_value_renderer/apply_unique_value_renderer_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/authenticate_with_oauth/authenticate_with_oauth_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/display_clusters/display_clusters_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/display_map/display_map_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/display_map_from_mobile_map_package/display_map_from_mobile_map_package_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/download_preplanned_map_area/download_preplanned_map_area_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/filter_by_definition_expression_or_display_filter/filter_by_definition_expression_or_display_filter_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/find_address_with_reverse_geocode/find_address_with_reverse_geocode_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/find_closest_facility_from_point/find_closest_facility_from_point_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/find_route/find_route_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/identify_layer_features/identify_layer_features_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/query_feature_table/query_feature_table_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/query_table_statistics/query_table_statistics_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/search_with_geocode/search_with_geocode_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/select_features_in_feature_layer/select_features_in_feature_layer_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/set_basemap/set_basemap_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/show_device_location/show_device_location_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/show_device_location_history/show_device_location_history_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/show_magnifier/show_magnifier_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/show_service_area/show_service_area_sample.dart';
+import 'package:arcgis_maps_sdk_flutter_samples/samples/style_point_with_simple_marker_symbol/style_point_with_simple_marker_symbol_sample.dart';
+import 'package:flutter/material.dart';
+
+// A list of all the Widgets for individual Samples.
+// Used by the Sample Viewer App to display the Widget when a sample is selected.
+// The key is the directory name for the sample which is in snake case. E.g. display_map
+const Map<String, Widget> sampleWidgets = {
```

Comment:
I think we prefer `const sampleWidgets = <String, Widget>{` or however that works